File: base.java8/src/main/java/alvin/base/java8/MainActivity.java
package alvin.base.java8;

import android.os.Bundle;
import android.widget.Button;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    private TextView textResult;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        this.textResult = findViewById(R.id.text_result);

        Button btnClickMe = findViewById(R.id.btn_click_me);
        btnClickMe.setOnClickListener(view -> this.textResult.setText(R.string.string_hello_world));
    }
}

File: ui.style/src/main/java/alvin/ui/style/main/views/MainActivity.kt
package alvin.ui.style.main.views

import alvin.ui.style.R
import alvin.ui.style.shape.views.ShapeActivity
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*
import org.jetbrains.anko.intentFor
import org.jetbrains.anko.sdk25.coroutines.onClick

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_main)
        setupActions()
    }

    private fun setupActions() {
        arrayOf(btn_shape,
                btn_selector,
                btn_layer_list,
                btn_animation,
                btn_style).forEach {
            it.onClick { btn ->
                startActivity(when (btn) {
                    btn_shape -> intentFor<ShapeActivity>()
                    else -> throw IllegalArgumentException()
                })
            }
        }
    }
}
